app/ios/OpenPassport/Info.plist

```diff
@@ -24,93 +24,75 @@
 	<string>2.5.2</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>proofofpassport</string>
+			</array>
+		</dict>
+	</array>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
 	<key>LSApplicationCategoryType</key>
-	<string />
+	<string></string>
 	<key>LSRequiresIPhoneOS</key>
-	<true />
+	<true/>
 	<key>NFCReaderUsageDescription</key>
 	<string>Need NFC to read Passport</string>
-	<key>NSAppTransportSecurity</key>
-	<string />
   <!-- Do not change NSAllowsArbitraryLoads to true, or you will risk app rejection! -->
 	<key>NSAllowsArbitraryLoads</key>
-	<false />
+	<false/>
 	<key>NSAllowsLocalNetworking</key>
-	<true />
-	<key>ITSAppUsesNonExemptEncryption</key>
-	<false />
+	<true/>
+	<key>NSAppTransportSecurity</key>
+	<string></string>
 	<key>NSCameraUsageDescription</key>
 	<string>Needed to scan the passport MRZ.</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Needed to secure the secret</string>
 	<key>NSHumanReadableCopyright</key>
-	<string />
+	<string></string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string />
+	<string></string>
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>We need access to your photo library to allow you to choose passport photos or save generated QR codes.</string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Advercase-Regular.otf</string>
+		<string>DINOT-Medium.otf</string>
+	</array>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
+	<true/>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>fetch</string>
+		<string>remote-notification</string>
+	</array>
 	<key>UILaunchStoryboardName</key>
 	<string>LaunchScreen</string>
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
 		<string>arm64</string>
 	</array>
+	<key>UIStatusBarHidden</key>
+	<false/>
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
-	<false />
+	<false/>
 	<key>com.apple.developer.nfc.readersession.iso7816.select-identifiers</key>
 	<array>
 		<string>A0000002471001</string>
 		<string>A0000002472001</string>
 		<string>00000000000000</string>
 	</array>
-	<key>CFBundleURLTypes</key>
-	<array>
-		<dict>
-			<key>CFBundleURLSchemes</key>
-			<array>
-				<string>proofofpassport</string>
-			</array>
-		</dict>
-	</array>
-	<key>UIAppFonts</key>
-	<array>
-		<string>Advercase-Regular.otf</string>
-		<string>DINOT-Medium.otf</string>
-		<string>Inter-Black.otf</string>
```

Comment:
xcode updated the order of the params, verified line by line that they are the same.

the only thing we need from t his update is the removal of the unused fonts